ci/travis_script_python.sh

```diff
@@ -14,12 +14,16 @@
 
 set -e
 
+source $TRAVIS_BUILD_DIR/ci/travis_install_conda.sh
+
 PYTHON_DIR=$TRAVIS_BUILD_DIR/python
 
 # Re-use conda installation from C++
 export MINICONDA=$HOME/miniconda
 export PATH="$MINICONDA/bin:$PATH"
-export PARQUET_HOME=$MINICONDA
+
+export ARROW_HOME=$ARROW_CPP_INSTALL
+export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$ARROW_CPP_INSTALL
```

Comment:
Not so happy that this is needed now but I can live with it as the `manylinux1`  `pyarrow` packages will not need it due to some separate post-processing.